src/javascript/app_2/Services/ws-methods.js

```diff
@@ -73,6 +73,14 @@ const WS = (() => {
     const subscribeProposalOpenContract = (contract_id = null, cb, should_forget_first) =>
         SubscriptionManager.subscribe('proposal_open_contract', { proposal_open_contract: 1, subscribe: 1, ...(contract_id && { contract_id }) }, cb, should_forget_first);
 
+    const subscribeProposalOpenContractAfterBuy = (buy_request) =>
```

Comment:
@OskarAhl IMO, it'd be better to change the function name, it conveys that we should call it after sending buy response.

Reply:
Would `subscribeProposalOpenContractOnBuy` be more clear?